docs/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = pipenv run sphinx-build
```

Comment:
Why this change?

Reply:
My bad, this shouldn't have been committed!